graphs using dictionary

```diff
@@ -0,0 +1,32 @@
+thisdict = {
+    }
+    
+thisdict[1]=[[2,6],[3,7]]#first number = room number. second = distance of connection
+thisdict[2]=[[1,6]]
+thisdict[3]=[[1,7]]
+print(thisdict)
+
+def findifconnect(room1,room2):
+    x = thisdict[room1]
+    for i in range(len(x)):
+        if x[i][0] == room2:
+            return True
+    return False
+
+def finddist(room1,room2):
+    x = thisdict[room1]
+    for i in range(len(x)):
+        if x[i][0] == room2:
+            return x[i][1]
+def addconnection(room1,room2,dist):
+    thisdict[room1].append([room1,dist])
+    thisdict[room2].append([room2,dist])
+
```

Comment:
'''def addconnection(room1,room2,dist):
    thisdict[room1].append([room2,dist])
    thisdict[room2].append([room1,dist])'''
Should it not be this (room1 changed to room 2 and vice-versa)